packages/graphql/src/schema.ts

```diff
@@ -29,6 +30,8 @@ export const graphqlSchema = makeSchema({
     },
   },
   plugins: [
+    nexusSlowGuardPlugin,
+    nexusDebugLogPlugin,
```

Comment:
ditto with other feedback on performance, should we not run these if the debug flags are not enabled?